### PR TITLE
web toy

### PR DESCRIPTION
**Pull request by bethanyhaubert**

### CODE REVIEW

**Comment by bethanyhaubert** [comment]
```
<p>This is Gita. </p>
<p><img src="../images/gita.jpg" width="500" height="400" alt="This is a picture of my cat, Gita."></p>

```

I added an images file into my 'main' folder under 'views' folder and added images to my favorites.erb page, but the photo isn't showing and I'm getting the following error message from sinatra when I click on the "../images/gita.jpg" link in the source code: 

```
# in main.rb
class MyApp
  get '/images/gita.jpg' do
    "Hello World"
  end
end

```

Do I need to create a new method within the MyApp class to display images? I also tried moving the images folder to just inside the 'views' folder instead of inside 'main' which is inside 'views'.

File: views/main/favorites.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<!-- error msg: 
+# in main.rb
+class MyApp
+  get '/images/gita.jpg' do
+    "Hello World"
+  end
+end
+ -->
+<p>This is Gita. </p>
+<p><img src="../images/gita.jpg" width="500" height="400" alt="This is a picture of my cat, Gita."></p>
```
Comment (sumeetjain) [inline views/main/favorites.erb:15]: You can make a folder in the project's root directory called **public** and put images there.

Then you can reference them with the `img` tag like `<img src="/gita.jpg"...`. Basically Sinatra takes everything in **public** and makes it available automatically from the root path. No controller actions needed to get things from the **public** directory.

**Comment by sumeetjain** [comment]
:+1:

**Comment by bethanyhaubert** [comment]
thank you!

**Pull request closed without merging**